src/plugins/intel_gpu/tests/test_cases/scatter_nd_update_gpu_test.cpp

```diff
@@ -808,7 +1059,7 @@ TEST(scatter_nd_update_gpu_fp16_test7, data5_indice2_update4) {
 TEST(scatter_nd_update_gpu_fp16_test6, data4_indice2_update3) {
```

Comment:
Is there any reason that we cannot cover these legacy tests with the new one?

Reply:
they can be covered with the new one, but I thought leaving the original test cases would be better for later purposes. Unlike the new test cases (TEST_P), the original ones(TEST) show all the values for data, indices, updates, expected results, so I thought I'd leave them for reference